xmtp_api_d14n/src/compat/d14n.rs

```diff
@@ -165,11 +158,6 @@ where
         &self,
         request: QueryGroupMessagesRequest,
     ) -> Result<QueryGroupMessagesResponse, Self::Error> {
-        self.message_client
```

Comment:
going to remove stats for d14n for now, i think there's a way to do it directly in the client since we have endpoint information in Query and Client request

Reply:
Yeah, that seems fine.